app/import_functions_test.go

```diff
@@ -2501,9 +2501,34 @@ func TestImportimportMultiplePostLines(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, 0, errLine)
 
+	// Create a pinned message.
+	data = LineImportWorkerData{
+		LineImportData{
+			Post: &PostImportData{
+				Team:     &teamName,
+				Channel:  &channelName,
+				User:     &user2.Username,
+				Message:  ptrStr("Pinned Message"),
+				CreateAt: ptrInt64(model.GetMillis()),
+				IsPinned: ptrBool(true),
+			},
+		},
+		1,
+	}
+	errLine, err = th.App.importMultiplePostLines(th.Context, []LineImportWorkerData{data}, false)
+	assert.Nil(t, err)
```

Comment:
```suggestion
	require.Nil(t, err)
```

Reply:
Hello @isacikgoz sounds good to me, I'll update the PR